.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: python
+python: 2.7
```

Comment:
ok, I'll fix that; anyway it was working as you can see here:

https://travis-ci.org/plone/plone.app.ldap/jobs/70307068#L94